mcos/de_noising/de_noise.py

```diff
@@ -0,0 +1,179 @@
+from sklearn.neighbors import KernelDensity
```

Comment:
Could we remove this file up a directory? That way we don't have to import
```
from mcos.de_noising.de_noise import de_noise_covariance_matrix
```
and can just do
```
from mcos.de_noise import de_noise_covariance_matrix
```

Reply:
for sure!